api/graphql/opening_hours/opening_hours_types.py

```diff
@@ -144,11 +145,8 @@ def resolve_opening_hours(self, info, **kwargs):
                         end_time=time.end_time.time(),
                         state=time.resource_state,
                         periods=time.periods,
-                        is_reservable=opening_hours_client.is_resource_open_for_reservations(
-                            str(self.hauki_resource_origin_id),
-                            time.start_time,
-                            time.end_time,
-                        ),
+                        is_reservable=ResourceState(time.resource_state)
```

Comment:
Is `opening_hours_client.is_resource_open_for_reservations` function still needed somewhere? I checked the code and looks like it is used, but does the same logic apply in other places?

Just wondering if we could remove it and replace with this simple state check 🤔

Reply:
yes, it used when validating the reservation creation (via scheduler)